src/modules/allItemsCounter.js

```diff
@@ -0,0 +1,5 @@
+import foodItems from './foodItems.js';
+
+const itemsCounter = () => foodItems.length;
+
+export default itemsCounter;
```

Comment:
- [ ] Great work team. I acknowledge the efforts put into this project. I noticed one thing though, the number of items rendered on the **_Home page_** of the application is `24` but the item counter displays `10`. You are counting the number of items in the `foodItems` array, but you ought to calculate the number of items rendered in the **_Home Page_**. Kindly fix this error and update the test for the item counter on the Home page. 👍 😎